R/filter_gene.R

```diff
@@ -17,11 +23,14 @@
 #' @import utils
 #' @export
 #'
-filter_gene <- function(geneExp,filter,threshold=NULL) {
+filter_gene <- function(expr,
+                        filter,
+                        threshold=NULL,
+                        expr_file_out = 'expression.txt') {
```

Comment:
Differentiating between reading/writing expression files seemed important.  Arguments to write expression files are called `expr_file_out`.  Arguments to read expression files align to the netZooR standard `expr_file`